src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -253,11 +253,20 @@ int net_device_table_mgr::map_net_devices()
 			continue;
 		}
 
+		int iftype = get_iftype_from_ifname(ifa->ifa_name);
+		if (iftype == ARPHRD_INFINIBAND) {
+			if (!check_ib_device_supported(ifa->ifa_name) || !verify_ipoib_mode(ifa)) {
+				continue;
+			}
+		} else {
+			if (!check_qp_creation(ifa->ifa_name, IBV_QPT_RAW_PACKET)) {
```

Comment:
we used to check capabilities by reading the file "/sys/module/ib_uverbs/parameters/disable_raw_qp_enforcement". I removed that part since it doesn't exist in inbox drivers. I'll look for a new way to check.